app/controllers/users_controller.rb

```diff
@@ -0,0 +1,31 @@
+class UsersController < ApplicationController
+  include UsersHelper
```

Comment:
- [ ] Rails helpers are mainly used to extract logic out of views. I suggest that you always have the params method in the controller as a private method.

Reply:
This method is suggested by the tutorial, we just followed it and we find it better to do it there instead of putting it inside the controller.